src/webui/abstractwebapplication.cpp

```diff
@@ -354,6 +354,7 @@ bool AbstractWebApplication::sessionStart()
         sessions_[session_->id] = session_;
 
         QNetworkCookie cookie(C_SID, session_->id.toUtf8());
+        cookie.setHttpOnly(true);
```

Comment:
I think it's going to make qBittorrent stop working if you have HTTPS connection configured.